src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -144,6 +153,34 @@ function MoneyRequestView({report, parentReport, policyCategories, shouldShowHor
         }
     }
 
+    /**
+     * Returns am array of components, one for each violation message
+     * @example
+     * [<View key={`amount.perDayLimit`}>
+     *    <Text style={[styles.ph5, styles.textLabelError]}>{"The amount exceeds the per-day limit"}</Text>
+     *  </View>,
+     *  //...
+     *  ]
+     * @type {function({field: string}): React.Node[]}
+     */
+    const ViolationMessages = useCallback(
```

Comment:
Yeah I definitely think this should be its own component. Other than stylistic consistency, this component is long and this doesn't need to live in here.

Reply:
k will do